Automattic/wp-calypso
# Jetpack Backup: Add feature flag for on demand backups

Resolves https://github.com/Automattic/jetpack-backup-team/issues/373

## Proposed Changes

* Add `jetpack/backup-on-demand` feature flag for development environment only
* Add Jetpack Cloud colors to the button
* Add BackupNowButton in backup main page under the feature flag

## Screenshot

![CleanShot 2024-02-09 at 19 30 07@2x](https://github.com/Automattic/wp-calypso/assets/1488641/028d1ec2-3d2b-4a82-9813-4833153244ee)

## Known issue

The new button looks different to "Copy Site" and has no margins, but we will address styles in a next PR.

## Testing Instructions



* Spin up a Jetpack Cloud live branch.
* Navigate to the Backup page
* Ensure you don't see the `Backup Now` button.
* Now try passing the `jetpack/backup-on-demand` feature flag via URL, like `?flags=jetpack/backup-on-demand" and reload the page.
* Ensure you see the `Backup Now` button.

## Pre-merge Checklist
